test/models/user_test.rb

```diff
@@ -736,4 +736,18 @@ class UserTest < ActiveSupport::TestCase
   test '.users_job returns all users when invalid job is passed' do
     assert_equal User.all, User.users_job('destroy_all')
   end
+
+  test 'clear_github_data should clear GitHub related fields' do
```

Comment:
```suggestion
  test '#clear_github_data' do
```

モデルのテストで、代表的なテストだけだったら他と一緒でこういう名前でいいかもです。

Reply:
ご指摘ありがとうございます。

``` ruby
test '#clear_github_data' do
```

と修正いたしました。